src/LookupTable/contexts/LookUpTableImportCsvContext.tsx

```diff
@@ -48,8 +49,7 @@ const LookUpTableImportCsvReducer = (
     case LookUpTableImportCsvActions.ImportCSV:
       return { ...state, file: action.payload }
     case LookUpTableImportCsvActions.SetTableName: {
-      const restrictCharacters = action.payload.replace(/[^A-z_]/gm, '')
-      return { ...state, tableName: restrictCharacters }
+      return { ...state, tableName: action.payload }
```

Comment:
This restriction not necessary. The "name" can be anything and it will automatically be converted to a singular snake_case name for the table by the back-end.